### DISCUSSION TITLE
Bulk/Multi-row/Selected-row Edit

### DISCUSSION THREAD

**Discussion by caopengau** [KevinVandy/mantine-react-table#235] [Ideas]
Any plan to add feature for Bulk/Multi-row/Selected-row Edit

**Comment by KevinVandy** [comment]
Maybe, would like to see some lightweight proposals

**Comment by caopengau** [comment]
It could be added to MRT_ToolbarAlertBanner as a button to open a modal, on the right side of the text "x of y row(s) selected".
On the opened modal, it will have a column picking dropdown, then another text input/textarea to capture value. At the bottom of the modal, It will cancel and confirm button.
